protocol/streaming/full_node_streaming_manager.go

```diff
@@ -152,12 +165,24 @@ func (sm *FullNodeStreamingManagerImpl) Subscribe(
 			sm.nextSubscriptionId,
 		)
 	}
+	for _, subaccountId := range subaccountIds {
```

Comment:
nit: can use `sIds` instead of deref'ing pointers

Reply:
done